source/Outfit.cpp

```diff
@@ -232,8 +239,22 @@ void Outfit::Add(const Outfit &other, int count)
 		else
 			oit->second += count * it.second;
 	}
+	for(const auto &it : other.reverseFlareSprites)
+	{
+		auto oit = reverseFlareSprites.begin();
```

Comment:
use `std::find_if` to get an iterator to an existing flare sprite.

Probably you can refactor this & the regular flare sprite code to use a single method.

Reply:
Can't seem to do any of that without breaking something.